.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+*Use the template below when reporting bugs. Please, make sure that
+you're running the latest stable version  and that the problem
+you're reporting hasn't been reported (and potentially fixed) already.*
+
+*When requesting new features or improvements to existing features you can
+discard the template completely. Just make sure to make a good case for your
+request.*
+
+**Remove all of the placeholder text in your final report!**
+
+## Expected behavior
+
+## Actual behavior
+
+## Steps to reproduce the problem
+
+*This is extremely important! Providing us with a reliable way to reproduce
+a problem will expedite its solution.*
+
+## Environment & Version information
+
+###  version information
+
+### CIDER version information
```

Comment:
I guess that'd be kind of optional for haystack, as one can use it without CIDER.